apisupport.ant/src/org/netbeans/modules/apisupport/project/ui/Bundle.properties

```diff
@@ -75,7 +75,6 @@ HINT_suite_project_root_node=Module suite project in {0}
 LBL_jnlp_master=JNLP Descriptor
 
 Templates/Licenses/license-cddl-netbeans-sun.txt=NetBeans CDDL/GPL
-Templates/Licenses/license-apache20-netbeans.txt=NetBeans Apache License 2.0
```

Comment:
Should be fixed this.

Reply:
That line is deleted.

Reply:
Yes, should be fixed in the same module. i.e. Please don't delete this line. (Please don't change projectui.)

Reply:
Not sure I understand. This line was used by the template registration in `layer.xml` but since I've deleted the template registration from this module the line is no longer relevant.